### DISCUSSION TITLE
pipenv install waited too long to install

### DISCUSSION THREAD

**Discussion by zhenzi0322** [pypa/pipenv#5510] [Q&A]
![image](https://user-images.githubusercontent.com/34839719/204469805-81090935-29b4-411d-9744-15f63a117c3a.png)

pipenv install I've been waiting for more than 10 minutes, and I haven't seen any module libraries start to install.

**Comment by kalebmckale** [comment]
Depending on the number of dependencies, the version specs, and the combination of dependencies, locking can take quite a while since it's running a graph-theoretic algorithm to find the best combination of package versions to ensure compatibility. Do you mind sharing your install line or Pipfile?
